Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -959,6 +952,11 @@ void SourceBuffer::evictCodedFrames(size_t newDataSize)
 
     MediaTime rangeStart = buffered.start(0);
     MediaTime rangeEnd = rangeStart + thirtySeconds;
+
```

Comment:
Why is this block needed?

Reply:
Just to be safe in case something else triggers samples leak.